fractal_tasks_core/tasks/cellpose_segmentation.py

```diff
@@ -338,14 +333,10 @@ def cellpose_segmentation(
         use_masks = False
     logger.info(f"{use_masks=}")
 
-    # Read pixel sizes from zattrs file
-    full_res_pxl_sizes_zyx = extract_zyx_pixel_sizes(
-        f"{zarrurl}/.zattrs", level=0
-    )
+    # Read pixel sizes from Zarr attributes
```

Comment:
Can we take this and the next part up to line 270 with the rest of the metadata extraction? (open to discuss whether it makes sense to group those)

Reply:
Note: it's clear that we could always push things a bit further, and also move the calculation of the Z/{X,Y} anisotropy into a method of the NGFF models.

Reply:
That's great! Getting anisotropy does seem fairly task specific (something cellpose needs to know, but not many other tasks), while getting pixel sizes and doing homogeneity checks is great to have in the NGFF models